thor/src/protocols/splice.rs

```diff
@@ -43,13 +42,13 @@ pub struct Message2 {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug)]
 pub struct Message3 {
-    splice_transaction_signature: Signature,
+    sig_TX_splice_TX_f_input: Signature,
```

Comment:
```suggestion
    sig_tx_splice_tx_f_input: Signature,
```

We just merged a patch that changes all `TX`s into `tx`s in identifiers (excl. const `TX_FEE`).